### DISCUSSION TITLE
Changes in project pending review for a long time

### DISCUSSION THREAD

**Discussion by YannMarti** [WalletConnect/walletconnect-monorepo#4415] [Developer Support]
Hey, i had the wrong store links in my wallet project(Legacy Wallet), changed it and it is pending review for 2 weeks now. Is it normal that review takes this long? It says reviews should be handled within 3 days, or is that just for new projects?

**Comment by glitch-txs** [comment]
Hi could you share your project ID?

**Reply by YannMarti** [reply]
sure, 8141668a92aad85f9f9f4b00e59914061df0f189b5b11fb35a15dc2708bb4596

**Reply by YannMarti** [reply]
its reviewed no, thanks. How long does it take for the changes to be reflected in the explorer?

**Reply by glitch-txs** [reply]
already approved: https://explorer.walletconnect.com/legacy-wallet

**Reply by YannMarti** [reply]
Still shows the old store links. I added a link for ios and fixed the play store link.

**Reply by YannMarti** [reply]
oh, it is now, thanks.
